calico-cloud/users/create-custom-role-for-entra-id-group.mdx

```diff
@@ -0,0 +1,55 @@
+---
+description: Create custom roles for Entra ID groups.
+---
+
+import IconUser from '/img/icons/user-icon.svg';
+
+# Create a custom role for an Entra ID group
+
+If you have Microsoft Entra ID configured as your identity provider, you can assign custom roles to Entra ID (formerly Azure AD) security group.
```

Comment:
"If you have Microsoft Entra ID configured as your identity provider" should be called out as the first prerequisite, and should include a link to `/users/user-management#add-your-own-identity-provider`.

Reply:
I had a prerequisite for contacting support that somehow got removed in the final version. I'll get this fixed.